files/jsbarcode/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Johan Lindell"
+github = "https://github.com/lindell/JsBarcode"
+homepage = "https://github.com/lindell/JsBarcode"
+description = "JsBarcode is a barcode generator written in JavaScript. It supports multiple barcode formats and it has no dependencies but works with jQuery if you are into that."
+mainfile = "JsBarcode.min.js"
```

Comment:
The file is named `JsBarcode.all.min.js` here https://github.com/lindell/JsBarcode/tree/v3.3.2/bin/browser